config.json

```diff
@@ -42,9 +43,9 @@
       "slug": "word-count",
       "topics": [
         "control_flow_loops",
-        "regular_expressions",
+        "lists",
         "maps",
-        "lists"
+        "regular_expressions"
```

Comment:
This ticket doesn't cover formatting the contents of other files. It's just to add the presubmit script.

Reply:
Revert this change